docs/troubleshoot/ze/troubleshoot-ze.md

```diff
@@ -44,7 +44,7 @@ To fix the configuration file causing the error:
 
 In some environments, the path for VS Code extensions (typically `~/.vscode/extensions`) can be read-only. This can happen when an environment has developers sharing a common read-only extensions folder that is managed by an admin with write access.
 
-In these cases, Zowe Explorer fails to activate because it cannot write to the `logs` and `temp` folders in the extension path. When Zowe Explorer launches, an error message displays. See the following examples.
+In these cases, Zowe Explorer fails to activate because it cannot write to the `logs` and `temp` folders in the extension path. When Zowe Explorer launches, an `EACCES: permission denied` error is displayed. See the following examples.
```

Comment:
We try to avoid passive tense. Might want to change it to "ana `EACCESS....` error displays."